### PR TITLE
Japari Library presence

### PR DESCRIPTION
**Pull request by FlashlightET**
A Kemono Friends wiki

### CODE REVIEW

File: Japari Library/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "id": "158418656861093888"
+  },
+  "url": "japari-library.com",
+  "description": {
+    "en_us": "Japari Library is a wiki based around the Kemono Friends multimedia franchise."
+  },
+  "service": "Japari Library",
+  "version": "1.0",
+  "logo": "https://japari-library.com/w/resources/assets/Jlibrarywglogo.png",
+  "thumbnail": "https://cdn.discordapp.com/attachments/544479934467276841/670661324732104704/favicon.png",
```
Comment (Bas950) [inline Japari Library/dist/metadata.json]: Upload the images to a site like imgur, so they are not on discords cdn or theirs.

File: Japari Library/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "author": {
+    "name": "Flashlight",
+    "id": "158418656861093888"
+  },
+  "url": "japari-library.com",
+  "description": {
+    "en_us": "Japari Library is a wiki based around the Kemono Friends multimedia franchise."
```
Comment (Bas950) [inline Japari Library/dist/metadata.json]: "en" not "en_us"

File: Japari Library/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  "description": {
+    "en_us": "Japari Library is a wiki based around the Kemono Friends multimedia franchise."
+  },
+  "service": "Japari Library",
+  "version": "1.0",
+  "logo": "https://japari-library.com/w/resources/assets/Jlibrarywglogo.png",
+  "thumbnail": "https://cdn.discordapp.com/attachments/544479934467276841/670661324732104704/favicon.png",
+  "color": "#4CC5FF",
+  "category": "anime"
+  "tags": ["anime", "wiki"]
```
Comment (Bas950) [inline Japari Library/dist/metadata.json]: You forgot a comma behind the category stuff, so the tags wont work...

**Comment by FlashlightET** [comment]
image
proof that it works, and updated metadata file

**Pull request merged**